django_x509/tests/test_ca.py

```diff
@@ -622,3 +622,15 @@ def test_datetime_to_string(self):
                          generalized_datetime.strftime(generalized_time))
         self.assertEqual(datetime_to_string(utc_datetime),
                          utc_datetime.strftime(utc_time))
+
+    def test_renew_certificate_key(self):
+        ca = self._create_ca()
```

Comment:
please generate 2 certs related to the CA and ensure that when the renew is triggered, also the related certs are renewed